CHANGELOG.md

```diff
@@ -4,12 +4,18 @@
 
 ### New features
 
-* Association methods that raise when the document is not found ([@pbrisbin][])
-
 ### Bug fixes
 
 ### Changes
 
+## v2.0.0 (2017-05-02)
+
+### New features
+
+* Association methods that raise when the document is not found ([@pbrisbin][])
```

Comment:
I _think_ that's more of a new feature. Now if you add a `!` to association methods and the associated resource isn't found, we'll throw a Doc not Found error.